Tests/ScipioKitTests/IntegrationTests.swift

```diff
@@ -105,9 +113,12 @@ final class IntegrationTests: XCTestCase {
                     .appendingPathComponent(destination)
                     .appendingPathComponent("\(frameworkName).framework")
 
-                let isPrivateFramework = frameworkName.hasPrefix("_")
-
-                if !isPrivateFramework {
+                if isClangFramework {
+                    XCTAssertTrue(
+                        fileManager.fileExists(atPath: frameworkRoot.appendingPathComponent("Headers/\(frameworkName).h").path),
+                        "\(xcFrameworkName) should contain an umbrella header"
+                    )
+                } else {
```

Comment:
Just a question. Does this `else` fall under the Mixed module?

Reply:
Currently, we don't have a test case for mixed modules. So current implementation, we don't care about them.
We should add the test case which contains both bridging headers and an umbrella headers.